docs-src/docs/reference/updateservice.md

```diff
@@ -11,6 +11,10 @@ The disadvantage of using this method is that you cannot perform A/B testing of
 
 **To replace a Deploy to refer to a new task definition:**
 
+<<<<<<< HEAD
```

Comment:
There are a few of these merge conflict markers still around in this file.

Reply:
My bad. Fixing now